css-box-model/styles.css

```diff
@@ -0,0 +1,11 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  margin: 12px;
```

Comment:
this will apply a margin of 12px in all directions, what properties do we need to apply to just left and right?